README.md

```diff
@@ -1,10 +1,40 @@
-# TUF repository automation demo
+# TUF repository demo
 
 This is a live [TUF](https://theupdateframework.io/) repository, intended for
 testing and demonstrations. It's
 maintained with  [TUF-on-CI](https://github.com/theupdateframework/tuf-on-ci).
 
-TUF clients can access the repository in https://jku.github.io/tuf-demo:
+The goals of this project are:
+* Maintain a long term repository with daily changes that the TUF community can use as a
```

Comment:
What do you mean by daily changes? If someone decides to push an artifact for instance, that we should complete the signing procedure in a day or that timestamp refreshes each day?